src/Backend/Modules/Faq/Layout/Templates/Edit.html.twig

```diff
@@ -97,35 +97,29 @@
           <small class="form-text text-muted">{{ 'msg.FeedbackInfo'|trans }}</small>
         </div>
         {% if feedback %}
-          <div class="card-group" id="feedback" role="tablist" aria-multiselectable="false">
-            {% for _itr_11 in feedback %}
-              <div class="card card-default">
-                <div class="card-heading" role="tab" id="_itr_11-heading-{{ _itr_11.id }}">
-                  <h4 class="card-title">
-                    <a data-toggle="collapse" data-parent="#_itr_11" href="#_itr_11-collapse-{{ _itr_11.id }}" aria-expanded="true" aria-controls="_itr_11-collapse-{{ _itr_11.id }}">
-                      {{ macro.icon('caret-right fa-fw') }}{{ _itr_11.text|truncate(150) }}
-                    </a>
-                  </h4>
-                </div>
-                <div id="_itr_11-collapse-{{ _itr_11.id }}" class="card-collapse collapse" role="tabpanel" aria-labelledby="_itr_11-heading-{{ _itr_11.id }}">
-                  <div class="card-body">
-                    {{ _itr_11.text }}
-                  </div>
-                  {% if showFaqDeleteFeedback %}
-                    <div class="card-footer">
-                      <div class="btn-toolbar">
-                        <div class="btn-group pull-left" role="group">
-                          {{ macro.buttonIcon('#', 'trash-alt', 'lbl.Delete'|trans|ucfirst, 'btn-danger', 'data-id=' ~ _itr_11.id ~ ' data-role=delete-feedback') }}
+          <div id="feedback" role="tablist" aria-multiselectable="false">
```

Comment:
I'm not sure this role attribute is used correctly here. A quick search learned me that a `role="tablist"` should have children with `role="tab"`. You should probably look into the aria-attribute too. I'm not an expert, but it looks like you just copied it without thinking about it ;)

Reply:
Indeed it was already there, I just changed some code in between and didn't pay attention to it, nice catch!